src/modules/Authentictation/Components/ResetPassword/ResetPassword.tsx

```diff
@@ -1,7 +1,224 @@
+import { AUTH_URLS, axiosInstance } from "../../../../services/urls";
+import CircularProgress from "@mui/material/CircularProgress";
+import { toast } from "react-toastify";
+import CustomPasswordInput from "../../../Shared/Components/CustomPasswordInput/CustomPasswordInput";
+import { ResetPasswordFormData } from "../../../../services/interfaces";
+const ResetPassword = () => {
+  const location = useLocation();
+  const myLocation = location.state;
+
+  const navigate = useNavigate();
+
+  const {
+    formState: { errors, isSubmitting },
+    register,
+    handleSubmit,
+    watch,
+  } = useForm<ResetPasswordFormData>({ defaultValues: { email: myLocation } ,mode : "onChange"});
+  const validationRules = getValidationRules(watch);
+
+  const onSubmitHandler = async (data: ResetPasswordFormData) => {
+    await axiosInstance
+      .post(AUTH_URLS.resetPassword, data)
+      .then((response) => {
+        console.log(response?.data?.message);
+        toast.success(
+          response?.data?.message || "password reset successfully !"
+        );
+        navigate("/login" , {state : data.email});
+      })
+      .catch((error) => {
+        console.log(error);
+        toast.error(
+          error?.response?.data?.message ||
+            "something went wrong please try again"
+        );
+      });
+  };
 
-export default function ResetPassword() {
   return (
-    <div>ResetPassword</div>
-  )
-}
+    <>
+      <Box
+        component="div"
+        sx={{
+          ml: { md: "7rem", xs: "2rem" },
+          mt: { md: "3.9rem", xs: "2rem" },
+        }}>
+        <Typography component="h1" sx={{ fontSize: "1.87rem" }}>
+          Reset Password
+        </Typography>
+
+        <Box component="div" mt="22px">
+          <Typography component="p">
+            If you already have an account register
+            <Typography component="span" display={"block"}>
+              You can{" "}
+              <Box
+                ml={1}
+                sx={{ color: red[600] }}
+                fontWeight={600}
+                component={"span"}>
+                <Link className="formLinks" to="/login">
+                  Login here !
+                </Link>
+              </Box>
+            </Typography>
+          </Typography>
+        </Box>
+
+        <Box component="div" mt={"96px"}>
+          <form onSubmit={handleSubmit(onSubmitHandler)}>
+            <Box
+              sx={{
+                display: "flex",
+                flexDirection: "column",
+                width: { xs: "95%", sm: "80%" },
+                paddingBottom: "0.5rem",
+                my: "1rem",
+              }}>
+              <Typography
+                variant="subtitle1"
+                component="label"
+                htmlFor="name-textfield"
+                sx={{ color: "#152C5B", fontSize: "16px" }}>
+                {"Email"}
+              </Typography>
+              <TextField
```

Comment:
I suggest sticking with custom text field created to enable consistent design and make any changes easier.